pkg/test/resourcefixture/testdata/basic/vertexai/v1beta1/vertexaiindex/dependencies.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Replace ${PROJECT_NUMBER?} below with your desired project number.
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicyMember
+metadata:
+  name: iampolicymember-${uniqueId}
+spec:
+  member: serviceAccount:service-${projectNumber}@gcp-sa-aiplatform.iam.gserviceaccount.com
+  role: roles/storage.admin # required by vertex AI service agent to access test data
+  resourceRef:
+    apiVersion: storage.cnrm.cloud.google.com/v1beta1
+    kind: StorageBucket
+    # external: ${KCC_VERTEX_AI_TEST_BUCKET}
```

Comment:
Is it possible to create a bucket in the test, avoiding the need for an env var?

Reply:
Oops I forgot to uncomment this line. To test this resource, we first need to set up a GCS bucket and pre-upload some content to it. While I know we can create a GCS bucket during testing, I'm unsure if we can also populate it with content during the test.